src/backends/compose/compose_backend.py

```diff
@@ -135,8 +135,10 @@ def get_status(self, tank_index: int, service: ServiceType) -> RunningStatus:
             case _:
                 return RunningStatus.PENDING
 
-    def exec_run(self, tank_index: int, service: ServiceType, cmd: str, user: str = "root") -> str:
+    def exec_run(self, tank_index: int, service: ServiceType, cmd: str, user="bitcoin") -> str:
```

Comment:
in "images: unified alpine-based dockerfile" (https://github.com/bitcoin-dev-project/warnet/pull/261/commits/4dc71ff74d4267b9c3f45530c1490702bd81c188):

```suggestion
    def exec_run(self, tank_index: int, service: ServiceType, cmd: str, user: str = "bitcoin") -> str:
```

Reply:
taken